website.js

```diff
@@ -18,6 +18,7 @@ var clientID        = options.identity.clientId
 var secret          = options.identity.clientSecret
 var redirect        = options.identity.redirectUrl
 var io              = require('socket.io')(server);
+var ip;
```

Comment:
I wasn't sure about this either. Hadn't found any other way to get the IP without using express. I'll look for some more options to get the IP in the socket.